src/llmcompressor/modifiers/quantization/gptq/base.py

```diff
@@ -131,23 +132,29 @@ def validate_sequential_update(cls, value: bool) -> bool:
     def resolve_quantization_config(self) -> QuantizationConfig:
         config = super().resolve_quantization_config()
 
-        # Resolve config with `self.actorder`
+        def resolve_actorder(existing):
+            # sentinel default only overrides if existing is None
+            if self.actorder == Sentinel("STATIC"):
```

Comment:
Do you mind expanding on the motivation in using this class instead of just checking self.actorder == "STATIC" (seems to be the only usecase?)?

I think I get it but not 100%.

Reply:
Any value could be used as a sentinel. The only purpose of instantiating from the `Sentinel` class is to make it obvious to a reader that the value is being used as a sentinel

For example, using a string might imply to a user that string names of actorder types are are valid values. `Sentinel`s are also impossible to accidentally pass in as a caller.